.pre-commit-hooks.yaml

```diff
@@ -1,6 +1,14 @@
 ---
 - id: stylua
-  name: StyLua
+  name: StyLua (cargo)
+  description: An opinionated Lua code formatter
+  entry: stylua
+  language: rust
+  types:
+    - lua
+
+- id: stylua-system
+  name: StyLua (system)
   description: An opinionated Lua code formatter
   entry: stylua
   language: rust
```

Comment:
Should this not be set to system?

Reply:
Yep. Thanks! What a pointless diff I submitted. Hah